playbooks/azure/openshift-cluster/provisioning_vars.yml.example

```diff
@@ -0,0 +1,76 @@
+---
+# Variables that are commented in this file are optional; uncommented variables
+# are mandatory.
+
+# Default values for each variable are provided, as applicable.
+# Example values for mandatory variables are provided as a comment at the end
+# of the line.
+
+# ------------------------ #
+# Common/Cluster Variables #
+# ------------------------ #
+# Variables in this section affect all areas of the cluster
+
+# Deployment type must be specified.
+openshift_deployment_type: # 'origin' or 'openshift-enterprise'
+
+# specify a clusterid
+# This value is also used as the default value for many other components.
+openshift_azure_clusterid:  # e.g test
+
+# azure region
+# This value will instruct the plays where all items should be created.
+# Multi-region deployments are not supported using these plays at this time.
+openshift_azure_region:  # e.g. eastus
+
+# -------- #
+# ssh keys #
+# -------- #
+
+# Specify the key pair name here to connect to the provisioned instances.
+#openshift_azure_vm_admin_username: openshift
+openshift_azure_vm_ssh_public_keys:
+- path: "/home/{{ openshift_azure_vm_admin_username }}/.ssh/authorized_keys"
+  key_data: "ssh-rsa AAAAB3... {{ openshift_azure_vm_admin_username }}_key"
+
+
+# image settings
+openshift_azure_vm_base_image_name: rhel7.4_base
+openshift_azure_vm_base_image:
+  name: "{{ openshift_azure_vm_base_image_name }}"
+  resource_group: "{{ openshift_azure_resource_group_name }}"
+
+openshift_azure_vm_storage_container: "{{ openshift_azure_clusterid }}"
+openshift_azure_storage_account_name: "openshift{{ openshift_azure_clusterid }}"
+
+# --------- #
+# AMI Build #
```

Comment:
s/AMI/\<whatever the azure equivalent is>/

Reply:
Image, fixed.